requirements.txt

```diff
@@ -1,3 +1,4 @@
+launchable>=1.1.3
```

Comment:
The CLI may change its interface in the future. You have to fix the version to 1.x with `launchable >= 1.1.3, ~= 1.0`.

Reply:
Ah, I didn't know I can use both `>=` and `=~`. Thank you!

Reply:
@ninjinkun Fixed it so would you review the PR again?